R/python.R

```diff
@@ -868,10 +868,16 @@ py_str.default <- function(object, ...) {
 py_str.python.builtin.object <- function(object, ...) {
 
   # get default rep
-  str <- py_str_impl(object)
+  id <- py_str_impl(object)
 
   # remove e.g. 'object at 0x10d084710'
-  str <- gsub(" object at 0x\\w{4,}", "", str)
+  id <- gsub(" object at 0x\\w{4,}", "", id)
+
+  # get list of accesible attr of the python object
+  attr <- capture.output(str(object$`__dict__`))
```

Comment:
What happens if the object doesn't have a `__dict__`? (or are Python objects universally guaranteed to have a `__dict__` member?)

Reply:
Yeah, that would probably throw an error. 

They are not universally guaranteed to have `__dict__`. User defined classes will have, though, I think. But there are probably exceptions. 

Ill try to come up with a safer approach.